demo/standalone/Vagrantfile

```diff
@@ -29,18 +29,8 @@ else
   File.open(".discovery-token", 'w') { |file| file.write("#{discovery_token}") }
 end
 
-private_build = false
-if ENV["START_LOCAL_ROOKD"] == "1"
```

Comment:
I haven't solved that yet. we could save the docker image to a file and copy them over.